benchmarl/models/mlp.py

```diff
@@ -152,6 +166,8 @@ class MlpConfig(ModelConfig):
     norm_class: Type[nn.Module] = None
     norm_kwargs: Optional[dict] = None
 
+    num_feature_dims: int = 1
```

Comment:
Let's keep this `MISSING` as we so not want to put default values in python but in yaml